File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/CreateFilter.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * Create firewall rule for your ip
 */




public class CreateFilter   {
  private String filterType = null;

  private Integer port = null;

  public CreateFilter filterType(String filterType) {
    this.filterType = filterType;
    return this;
  }

  /**
   * Get filterType
   * @return filterType
   **/
    public String getFilterType() {
    return filterType;
  }

  public void setFilterType(String filterType) {
    this.filterType = filterType;
  }

  public CreateFilter port(Integer port) {
    this.port = port;
    return this;
  }

  /**
   * Get port
   * @return port
   **/
    public Integer getPort() {
    return port;
  }

  public void setPort(Integer port) {
    this.port = port;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    CreateFilter createFilter = (CreateFilter) o;
    return Objects.equals(this.filterType, createFilter.filterType) &&
        Objects.equals(this.port, createFilter.port);
  }

  @Override
  public int hashCode() {
    return Objects.hash(filterType, port);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class CreateFilter {\n");
    
    sb.append("    filterType: ").append(toIndentedString(filterType)).append("\n");
    sb.append("    port: ").append(toIndentedString(port)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/create_filter.dart
part of swagger.api;

class CreateFilter {
  
  String filterType = null;

  int port = null;

  CreateFilter();

  @override
  String toString() {
    return 'CreateFilter[filterType=$filterType, port=$port, ]';
  }

  CreateFilter.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    filterType = json['filter_type'];
    port = json['port'];
  }

  Map<String, dynamic> toJson() {
    return {
      'filter_type': filterType,
      'port': port
     };
  }

  static List<CreateFilter> listFromJson(List<dynamic> json) {
    return json == null ? new List<CreateFilter>() : json.map((value) => new CreateFilter.fromJson(value)).toList();
  }

  static Map<String, CreateFilter> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, CreateFilter>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new CreateFilter.fromJson(value));
    }
    return map;
  }
}
